src/schema/numeric_options.rs

```diff
@@ -17,6 +18,12 @@ pub struct NumericOptions {
     fieldnorms: bool, // This attribute only has an effect if indexed is true.
     fast: bool,
     stored: bool,
+    #[serde(skip_serializing_if = "is_false")]
```

Comment:
is there a reason to skip serialization? 
```suggestion
```

Reply:
just a little less noise when reading the meta.json